pkg/ethutil/ethutil_test.go

```diff
@@ -93,10 +93,7 @@ func TestEthUtilSuite(t *testing.T) {
 // This image starts an anvil node with the Rollups contracts already deployed.
 func newDevNetContainer(ctx context.Context) (testcontainers.Container, error) {
 	req := testcontainers.ContainerRequest{
-		Image: "cartesi/rollups-devnet:devel",
-		Env: map[string]string{
-			"ANVIL_IP_ADDR": "0.0.0.0",
```

Comment:
Removing the `ANVIL_IP_ADDR: 0.0.0.0` may cause problems to connect to it from other container.

I can't tell if this environment is being defined elsewhere, just ignore my comment if that's the case.